OpenRA.Mods.Common/Activities/Move/MoveWithinRange.cs

```diff
@@ -47,6 +47,14 @@ protected override IEnumerable<CPos> CandidateMovementCells(Actor self)
 			var maxCells = (maxRange.Length + 1023) / 1024;
 			var minCells = minRange.Length / 1024;
 
+			if (minCells != 0 && Target.IsInRange(self.CenterPosition, minRange))
+			{
+				// Do extra dot product check to prevent the precious artillary unit from charging into enemy.
```

Comment:
Nit: Its not really appropriate to mention artillery (a mod specific unit) this deep in the generic RTS code.  There's also a spelling mistake!.

Can you please change this to something like "// Avoid cells on the far side of the enemy by requiring a negative dot product."?